src/index.spec.ts

```diff
@@ -30,13 +30,23 @@ describe('Try original simple-save-retrieve', () => {
     const val = tree.get('test');
     should.exist(val.value);
     val.value!.should.equal('one');
+    const val1 = tree.batchGet([Buffer.from('test')]);
+    should.equal(val.value, val1[0].value);
```

Comment:
In the future, please use a deep equals... val.value.should.deep.equals(). Also this belonged in another test.